pixart.js

```diff
@@ -1 +1,16 @@
+var setColors =document.querySelector(".color");
+var colorField =document.querySelector(".square");
+var brush = document.querySelector(".brush");
 
+$("button")[0].addEventlistener("clicks", handleClickEvent);
+$("colorField")[0].addEventlistener("clicks", handleClickEvent);
+
+for (var i = 0; i < 20; i ++){
+  var = div.createElement("div");
+}
+
+// function to click and change color
+function colorChange(){
+  document.getElementById('change').onclick = colorChange;
```

Comment:
There is no element in the document with an id of "change".  I don't see a variable called `colorChange`.